app/models/moderator.ts

```diff
@@ -0,0 +1,17 @@
+import DS from 'ember-data';
+
+import OsfModel from './osf-model';
+
+const { attr } = DS;
+
+export default class ModeratorModel extends OsfModel {
+    @attr('string') permissionGroup!: string;
```

Comment:
can we type this property with an enum and export it for use in other Moderator related stuffs?